src/app/views/App.tsx

```diff
@@ -328,6 +339,14 @@ class App extends Component<IAppProps, IAppState> {
       padding: 10
     };
 
+    const handleInstallTeamsAppButton = () => {
+      if (this.state.hideRSCPopupValue) {
+        this.setRSCPopupDontShowAgain();
+      }
+      this.toggleDialog();
+      window.open(TEAMS_APP_URL, '_blank')
+    }
+
```

Comment:
Currently only the Install button on the popup (the popup has Install, Learn more -- two buttons) will persist the localStorage flag if the Don't show again checkbox is checked. 

The Learn more button only opens RSC docs in a new tab and the popup will still be open when the user comes back to the Graph Explorer after viewing the docs.